server/build/build.py

```diff
@@ -229,6 +229,18 @@ def build_each_tool(dirname, section_name, des_build_file, arch='x86_86'):
     log_name = "%s.log" % section_name
     log_file = os.path.join(TMP_DIR, log_name)
     start_time = datetime.datetime.now()
+
```

Comment:
Suggestion: Opening files in append mode and deleting the required sections looks more logical.